pkg/server/grpc_server_test.go

```diff
@@ -1051,26 +1055,29 @@ func TestAPIWithGitLab(t *testing.T) {
 		t.Fatalf("unexpected length of leaf certificate URIs, expected 1, got %d", len(leafCert.URIs))
 	}
 
-	gitLabURL := fmt.Sprintf("https://gitlab.com/%s@refs/heads/%s", claims.ProjectPath, claims.Ref)
+	baseUrl := "https://gitlab.com/"
```

Comment:
the linter isn't happy with `baseUrl`, perhaps just switch to `baseURL`?

I'm about to cut a new release and roll out to staging so if we can turn this around quickly it can be included in that.

Reply:
```suggestion
	baseURL := "https://gitlab.com/"
```